### PR TITLE
Lesson-2: added tasks

### PR DESCRIPTION
**Pull request by NPaulGuy**

### CODE REVIEW

File: Lesson2/Task11.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * 	Создайте многомерный массив $arr. С его помощью выведите на экран слова 'joomla', 'drupal', 'зеленый', 'красный'. 
+ *  $arr = [
+ *	'cms'=>['joomla', 'wordpress', 'drupal'],
+ *	'colors'=>['blue'=>'голубой', 'red'=>'красный', 'green'=>'зеленый']
+ *  ];
+ */
+$arr = [
+	'cms'=>['joomla', 'wordpress', 'drupal'],
```
Comment (ckomarov) [inline Lesson2/Task11.php:10]: add spaces around operators

Comment (ckomarov) [reply Lesson2/Task11.php:10]: 'cms' => [...

File: Lesson2/Task13.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * 	Пусть теперь в переменной $lang хранится язык (она принимает одно из значений или 'ru', или 'en' - либо то, либо то), а в переменной $day - номер дня. 
+ * Выведите словом день недели, соответствующий переменным $lang и $day. То есть: если, к примеру, $lang = 'ru' и $day = 3 - то выведем 'среда'. 
+ */
+$lang = 'ru';
+$day = 5;
+$arr = [
+    'ru' => [1 => 'понедельник', 'вторник','среда','четверг','пятница', 'суббота','воскресенье'],
+    'en' => [1 => 'monday', 'tuesday','wednesday','thursday','friday', 'saturday','sunday']
```
Comment (ckomarov) [inline Lesson2/Task13.php:10]: why all arrays starts from 1?

File: Lesson2/Task6.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
+ *  Создайте массив $arr. Выведите на экран элемент с ключом 'c'. 
+ *  $arr = ['a'=>1, 'b'=>2, 'c'=>3];
+ */
+$arr = ['a'=> 1, 'b'=> 2, 'c'=> 3];
```
Comment (ckomarov) [inline Lesson2/Task6.php:6]: spaces around operators, psr-12

File: Lesson2/Task7.php [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
+ * Создайте массив $arr. Найдите сумму элементов этого массива. 
+ * $arr = ['a'=>1, 'b'=>2, 'c'=>3];
+ */
+$arr = ['a'=> 1, 'b'=> 2, 'c'=> 3];
```
Comment (ckomarov) [inline Lesson2/Task7.php:6]: same ^

File: Lesson2/Task8.php [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
+ * Создайте массив заработных плат $arr. Выведите на экран зарплату Пети и Коли. 
+ * $arr = ['Коля'=>'1000$', 'Вася'=>'500$', 'Петя'=>'200$'];
+ */
+$arr = ['Коля'=> '1000$', 'Вася'=> '500$', 'Петя'=> '200$'];
```
Comment (ckomarov) [inline Lesson2/Task8.php:6]: same ^

File: Lesson2/Task9.php [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Создайте ассоциативный массив дней недели. Ключами в нем должны служить номера дней от начала недели (понедельник - должен иметь ключ 1, вторник - 2 и т.д.). Выведите на экран текущий день недели. 
+ *
+ */
+$arr = [1 => 'понедельник', 'вторник','среда','четверг','пятница', 'суббота','воскресенье'];
```
Comment (ckomarov) [inline Lesson2/Task9.php:6]: why array starts from 1 again?

**Review by ckomarov — CHANGES_REQUESTED** [review]

File: Lesson2/Task3.php [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?php
+/**
+ * Создайте массив $arr=['a', 'b', 'c', 'd'] и с его помощью выведите на экран строку 'a+b, c+d'. 
+ */
+$arr = ['a','b','c','d'];
+echo "$arr[0]+$arr[1], $arr[2]+$arr[3]";
```
[the remark this answers is not in the document]
Comment (NPaulGuy) [reply Lesson2/Task3.php:6]: "$arr[0] + $arr[1] , $arr[2] + $arr[3]" works the same way.

**Pull request merged**